packages/vite-native-client/src/client.ts

```diff
@@ -1,3 +1,4 @@
+import { DEFAULT_PORT } from './../../vxrn/src/utils/constants';
```

Comment:
@natew - should I use this one, add a consts package or just create a consts on `vite-native-client`

Reply:
makes sense why build would break